import-io/python-lib/importio_utils.py

```diff
@@ -1,6 +1,13 @@
 # coding: UTF8
-import urlparse, requests, sys, dataiku, time
+import urlparse
+import requests
+import sys
+import dataiku
+import time
 from dataiku.customrecipe import *
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
Isn't basic config necessary?
I'd rather not use __name__ and not even use `logger =`
just use `logging.basicConfig` with a decent format (for example I probably did something not horrible for example in azure cognitive services plugins) and then just use `logging.info`

Reply:
+1 to Joachim, we should share config (nothing fancy, just simple stuff) across plugins)